src/promql.grammar

```diff
@@ -169,7 +169,6 @@ FunctionCallBody {
 FunctionCallArgs {
   FunctionCallArgs "," Expr |
   Expr |
```

Comment:
You'll need to remove the `|` from this line then.

Reply:
my bad sorry about it. It's fixed